docs/en_US/operation/manager-ui/overview.md

```diff
@@ -24,16 +24,16 @@ From eKuiper version 0.9.1, whenever a new version of eKuiper is released, the c
 
 ### Install eKuiper
 
-- Pull eKuiper's Docker image from [Docker Image Library](https://hub.docker.com/r/lfedge/ekuiper/tags). Since it is required to install the plugin in this article, you must use the `ekuiper:1.3.1-slim` image (`ekuiper:1.3.1-alpine` image is relatively small and easy to install, but due to the lack of some necessary library files, the plug-in cannot run normally. The `ekuiper:1.3.1` image is the development version, which is suitable for use in the development phase).
+- Pull eKuiper's Docker image from [Docker Image Library](https://hub.docker.com/r/lfedge/ekuiper/tags). Since it is required to install the plugin in this article, you must use the `ekuiper:1.4.2-slim` image (`ekuiper:1.4.2-alpine` image is relatively small and easy to install, but due to the lack of some necessary library files, the plug-in cannot run normally. The `ekuiper:1.4.2` image is the development version, which is suitable for use in the development phase).
```

Comment:
Do we need to update this version number for each release? Let's at least use 1.4-alpine version, so that we only need to update for major release